bin/audit-svg-colors.js

```diff
@@ -16,36 +16,69 @@ const { readFileSync } = require( 'fs' );
 const path = require( 'path' );
 const PALETTE = require( '@automattic/color-studio' );
 const chroma = require( 'chroma-js' );
-const _ = require( 'lodash' );
+
+/**
+ * Native Sort - replacement of ._sortBy Lodash function
+ *
+ * @returns Sorted array.
+ */
+const sortBy = () => {
+	return ( objA, objB ) => {
+		if ( objA.to.name > objB.to.name ) {
+			return 1;
+		} else if ( objB.to.name > objA.to.name ) {
+			return -1;
+		}
+		return 0;
+	};
+};
 
 /**
  * Palette color subsets
  */
 
+/**
+ * pickBy function is a replacement for the Lodash ._pickby
+ *
+ * @param {*} obj a palette of colors
+ * @param {*} fn This function returns truthy or falsy value to filter the colors
+ * @returns objext with filtered color values
+ * Used Object.keys() and Array.prototype.filter()to remove the keys for which fn returns a falsy value.
+ * Used Array.prototype.reduce() to convert the filtered keys back to an object with the corresponding key-value pairs.
+ */
+const pickBy = ( obj, fn ) =>
+	Object.keys( obj )
+		.filter( ( key ) => fn( obj[ key ], key ) )
+		.reduce( ( acc, key ) => ( ( acc[ key ] = obj[ key ] ), acc ), {} );
+
 // The subset of palette colors allowed in illustrations
-const PALETTE_ILLUSTRATION_COLORS = _.pickBy( PALETTE.colors, ( colorValue, colorName ) => {
-	// Avoid using pure black
-	if ( colorValue === '#000' ) {
-		return;
-	}
-	// Avoid specific colors for illustration use
-	return ! _.startsWith( colorName, 'Simplenote Blue' );
-} );
+const PALETTE_ILLUSTRATION_COLORS = PALETTE?.colors
```

Comment:
Are we ever going to have a use case where `PALETTE` won't be an object? I think it will always be an object and therefore the optional chaining might be unnecessary.